firebase-abt/src/main/java/com/google/firebase/abt/FirebaseABTesting.java

```diff
@@ -204,12 +204,8 @@ public void reportActiveExperiment(AbtExperimentInfo activeExperiment) throws Ab
   public void validateRunningExperiments(List<AbtExperimentInfo> runningExperiments)
       throws AbtException {
     throwAbtExceptionIfAnalyticsIsNull();
-    Set<String> runningExperimentIds = new HashSet<>();
-    for (AbtExperimentInfo runningExperiment : runningExperiments) {
-      runningExperimentIds.add(runningExperiment.getExperimentId());
-    }
     List<ConditionalUserProperty> experimentsToRemove =
-        getExperimentsToRemove(getAllExperimentsInAnalytics(), runningExperimentIds);
+        getExperimentsToRemove(getAllExperiments(), runningExperiments);
```

Comment:
not added by you, but `getAllExperiments` doesn't really indicate that it's from Analytics. Maybe just add a comment here to reduce confusion for our future selves

Reply:
yeah, the naming is a bit confusing. added some comments for now and maybe will revisit renaming the public method in another change.